source/acquaman/AMScanActionControllerBasicFileWriter.cpp

```diff
@@ -78,8 +78,17 @@ void AMScanActionControllerBasicFileWriter::writeToFile(int fileRank, const QStr
 		qDebug() << "Stream (1) writing differential " << startTime.msecsTo(endTime) << " for size " << textToWrite.size();
 		*/
 
+		QFileInfo dataFileInfo(filePath_+".dat");
+		int sizeBeforeWrite = dataFileInfo.size();
+
 		AMTextStream dataStream(dataFile_);
 		dataStream << textToWrite;
+		dataStream.flush();
```

Comment:
My understanding is the exact opposite.  Since QTextStream is specifically for handling strings, it automatically flushes the data to disk.  Hence why it says that calls to ```flush()``` are ignored.  Since you call ```QFileInfo::refresh()``` below, this shouldn't be necessary.